### PR TITLE
Vehicle equip fix2

### PR DESCRIPTION
**Pull request by FluffyRaKu**
Fixed unlimited vehicle ammo and added weapon abbreviations to the company management screen.

### CODE REVIEW

**Comment by FluffyRaKu** [comment]
Tested on my end after merging to latest version. Should be good to go.

File: scripts/scr_ui_manage/scr_ui_manage.gml
Comment (GalacticChimp) [inline scripts/scr_ui_manage/scr_ui_manage.gml:728]: commented out?

File: scripts/scr_ui_manage/scr_ui_manage.gml
Comment (GalacticChimp) [inline scripts/scr_ui_manage/scr_ui_manage.gml:738]: same?

File: scripts/scr_ui_manage/scr_ui_manage.gml
Comment (GalacticChimp) [inline scripts/scr_ui_manage/scr_ui_manage.gml:907]: please rename vars, it's very hard to understand what is going on with names likes ar_ge, ar_mb, x5 etc

**Review by GalacticChimp — CHANGES_REQUESTED** [review]
Overall looks fine, but please take a look at my comments.

File: scripts/scr_ui_manage/scr_ui_manage.gml
Comment (FluffyRaKu) [reply scripts/scr_ui_manage/scr_ui_manage.gml:907]: These variables are actually shared with the marine equip logic. ge is the wargear slot and mb is the mobility slot. Effectively wep1, wep2, armour, wargear and mobility are the equipment 5 slots, although they are remapped onto wep1, wep2, wep3, upgrade and accessory for vehicles.

I'll update the x5 though, as I don't think it's relied upon anywhere else in the code.

File: scripts/scr_ui_manage/scr_ui_manage.gml
Comment (FluffyRaKu) [reply scripts/scr_ui_manage/scr_ui_manage.gml:728]: The commented sections are inherited from Duke's original code. I'm not entirely sure why they are there in the first place (something about artifacts?), whether they are holdovers from an older version, an internal debug tool he used during development or a remnant from one of his incomplete additions before he abandoned the project.

They can obviously be removed, but I have currently left them in as they do provide some hits as to either how things used to work or possibly regarding intended future functionality and I didn't want to destroy potential documentation for others.

File: scripts/scr_ui_manage/scr_ui_manage.gml
Comment (FluffyRaKu) [reply scripts/scr_ui_manage/scr_ui_manage.gml:738]: As above, want them removing or leaving for potential documentation regarding Duke's original work?

File: scripts/scr_ui_manage/scr_ui_manage.gml
Comment (FluffyRaKu) [reply scripts/scr_ui_manage/scr_ui_manage.gml:907]: x5 now called "xoffset". x5 is referenced further down in the code, but it resets the variable as part of that section and doesn't have any direct relation to this part.

File: scripts/scr_ui_manage/scr_ui_manage.gml
Comment (GalacticChimp) [reply scripts/scr_ui_manage/scr_ui_manage.gml:738]: I recommend to remove them. We will still have them in git history, so we can always check Duke's old code.
But this is a small issue, your call.

**Review by GalacticChimp — APPROVED** [review]

File: scripts/scr_ui_manage/scr_ui_manage.gml
Comment (FluffyRaKu) [reply scripts/scr_ui_manage/scr_ui_manage.gml]: Tidied up white space in this section and added comments to detail mapping between vehicle and marine equipment.

File: scripts/scr_ui_manage/scr_ui_manage.gml
Comment (FluffyRaKu) [reply scripts/scr_ui_manage/scr_ui_manage.gml]: Duke's original comments removed and my own comments regarding marine-vehicle equipment mapping added.

**Comment by FluffyRaKu** [comment]
I think this particular set of changes should be good to go. Unless there's some other tweak that you can see that would help, feel free to merge whenever.

**Pull request merged**
